util/translate_fight.py

```diff
@@ -28,6 +31,145 @@
     'Weakness',
 ]
 
+# Setting the languages for use with XIVAPI/CafeMaker
+xiv_langs = ['en', 'de', 'fr', 'ja']
+xiv_lang_map = {'en': 'Name_en',
+                'de': 'Name_de',
+                'fr': 'Name_fr',
+                'ja': 'Name_ja',
+                }
+
+def find_timeline_translatables(args):
+    translatables = defaultdict(dict)
+    # These keys do double duty as the indices for XIVAPI's search function.
+    for element in ['action_ids', 'status', 'bnpcname', 'PlaceName']:
+        translatables[element] = []
+
+    with args.timeline as file:
+        for line in file:
+            # Ignore all commented lines
+            if not line[0].isdigit():
+                continue
+
+            # Boil each remaining line down to its sync regexes
+            cleaned_line = e_tools.clean_and_split_tl_line(line, False)
+
+            # If for some reason the line doesn't clean up properly, just skip it.
+            if cleaned_line is None:
+                continue
+
+            sync_match = e_tools.is_tl_line_syncmatch(cleaned_line)
+
+            # If we don't have a match for whatever reason, ignore the line.
+            if sync_match is None:
+                continue
+
+            # We don't want to catch the reset line.
+            if sync_match.group(1).startswith('is no longer sealed'):
```

Comment:
Why do you want to ignore this? Is there some larger category of things that should be ignored than just "is no longer sealed"? Also, why would you not include zone unsealing in terms of looking for zone names as you do with zone sealing below?

Reply:
That's a relic from when the 0.0 reset line was giving me problems. It looks like that shouldn't be an issue anymore, so you're definitely correct it's best to just remove it.

(In practice, since resets just sync to `/is no longer sealed/` now, it's moot, but you're right, there's no reason to avoid those lines in general, since we already de-duplicate entries in-line.)